test/notebooks/nbtest_itde.py

```diff
@@ -0,0 +1,22 @@
+from exasol.secret_store import Secrets
+from exasol.itde_manager import (
+    bring_itde_up,
+    take_itde_down
+)
+from exasol.connections import open_pyexasol_connection
+
+def test_itde(tmp_path):
+    store_path = tmp_path / 'tmp_config.sqlite'
+    store_password = "password"
+    secrets = Secrets(store_path, master_password=store_password)
+    bring_itde_up(secrets)
+    try:
+        con = open_pyexasol_connection(secrets)
+        try:
+            result = con.execute("select 1").fetchmany()
+            assert result[0][0] == 1
+        finally:
+            con.close()
```

Comment:
Would it be possible to use a `with` block with automatic closing of the connection?

Reply:
ah, good point, I wasn't sure if the pyexasol connection is context manager. it is, will change this